fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVPlanUtil.java

```diff
@@ -55,6 +57,9 @@ public static ConnectContext createMTMVContext(MTMV mtmv) {
         ctx.getState().reset();
         ctx.setThreadLocalInfo();
         ctx.getSessionVariable().allowModifyMaterializedViewData = true;
+        // Disable add default limit rule to avoid refresh data wrong
+        ctx.getSessionVariable().setDisableNereidsRules(
```

Comment:
If the default values are changed in the future, there may be issues here. Can we use an additional method

Reply:
the default value of `disableNereidsRules` session should always be empty ''